searching.rst

```diff
@@ -51,8 +51,9 @@ The various placeholders are as follows:
 
 :var:`name`:
   The name of the package to be located,
-  including both the proper case name,
-  and the name converted to lower case.
+  including both the case as specified
+  by the requester and the name converted
+  to lower case.
```

Comment:
Two minor points: first, I think "consumer" would be more accurate here¹, and second, ["try to break at 'natural' boundaries"](https://cps-org.github.io/cps/development.html#contributing).

```suggestion
  including both the case as specified by the consumer,
  and the name converted to lower case.
```

\- or -

```suggestion
  The name of the package to be located, including both
  the case as specified by the consumer,
  and the name converted to lower case.
```

Either is fine; use whichever you think expresses the more natural "thought break". Note that the second needs to also alter the preceding line, and can't be (correctly) applied via github. (FWIW, I'm slightly inclined to the latter.)

"user / consumer" would also be acceptable.

¹ We do use "user" in a few places. A glossary might be helpful; I *think* we're consistently using "user" to refer to the ultimate consumer, especially e.g. the point where a human has written `find_package` or invoked `cps-config`. A "consumer" includes transitive consumers, e.g. when a package depends on another package. Thus, I think "consumer" is superior here.